xml/System.Web.Handlers/ScriptModule.xml

```diff
@@ -104,17 +104,21 @@
         <Parameter Name="app" Type="System.Web.HttpApplication" Index="0" FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
       </Parameters>
       <Docs>
-        <param name="app">An <see cref="T:System.Web.HttpApplication" /> object that provides access to the methods, properties, and events that are common to all applications in an ASP.NET application.</param>
+        <param name="app">An [HttpApplication](https://msdn.microsoft.com/en-us/library/4wt3wttw\(v=vs.110\)) object that provides access to the methods, properties, and events that are common to all applications in an ASP.NET application.</param>
```

Comment:
Just being nosy - this looks to be changing a `<see>` link to Markdown - something which we've just been correcting in the opposite direction in https://github.com/dotnet/docs/issues/1931. Is this expected, as something to get the basic data in, and then fix later?

Reply:
I appreciate the feedback @ChrisMaddock! This PR shouldn't be modifying what we already have. Let me take a closer look.